### ISSUE TITLE
anchors don't auto scroll if ajax or setTimeout is used

### ISSUE THREAD

**Issue by Nicoduweb** [alvarotrigo/multiscroll.js#77]
Hello,

I'm trying to use multiscroll.js after ajax load html.
All seems to work except "autoscroll" on load or refresh.

``` javascript
$(document).ready(function() {
  $.get( "multiscroll.php", function( data ) {
    $( "#myContainer" ).html( data );
    $('#myContainer').multiscroll({
      anchors: ['un', 'deux', 'trois', 'quatre', 'cinq', 'six'],
    });
  });
});
```

The problem can be reproduced with setTimeout function like that  :

``` javascript
$(document).ready(function() {
  setTimeout(function(){
    $('#myContainer').multiscroll({
      anchors: ['un', 'deux', 'trois', 'quatre', 'cinq', 'six'],
    });
  }, 3000);
});
```

Am i doing something wrong ?

Thx

**Comment by alvarotrigo** [comment]
Feel free to create a pull request.
The solution seems to be the same one implemented in fullpage.js thanks to [this PR](https://github.com/alvarotrigo/fullPage.js/pull/1982)

**Comment by Nicoduweb** [comment]
Thank you !
I'll read this.

**Comment by alvarotrigo** [comment]
Fixed on the latest version multiScroll 0.2.3
